lcov.spec

```diff
@@ -1,12 +1,12 @@
-### RPM external lcov 1.9
+### RPM external lcov 1.15
 ## NOCOMPILER
-Source: http://heanet.dl.sourceforge.net/sourceforge/ltp/%{n}-%{realversion}.tar.gz
+Source: https://github.com/linux-test-project/%{n}/archive/refs/tags/v%{realversion}.tar.gz
 Patch0: lcov-merge-files-in-same-dir
 
 
 %prep
 %setup -n %{n}-%{realversion}
-%patch0 -p1
+%patch0 -p0
```

Comment:
this should be `-p1`

Reply:
No, it should be `-p0`, but I forgot to keep `bin` in the path.